Controller/TicketingController.php

```diff
@@ -38,8 +38,7 @@ public function addTicket(Request $request, TicketManager $ticketManager): \Symf
         if($ticketForm->isSubmitted() && $ticketForm->isValid()){
             $ticketManager->createTicket($user, $ticket);
             $this->addFlash('success', 'The ticket is online !');
-            return $this->render('@Ticketing/ticketing/index.html.twig', [
-            ]);
+            return $this->redirectToRoute('all_ticketing');
```

Comment:
Fix the identation

1 empty line before a return

and 0 after

```php
#my code here

return ...;
```

Reply:
CTRL+ALT+L on JetBrains IDE (PHPStorm) to fix the identation and syntax

Reply:
CTRL+ALT+MAJ+L for more details
![image](https://user-images.githubusercontent.com/13522273/40943109-51962a32-6850-11e8-9c0a-b1eb363c84d3.png)